minio/minio
# list objects in browser ordered by last modified

## Description

While listing the objects from the selected bucket on browser ui, 
we will pull all the objects of the bucket from backend and 
by default sort based on last modified date.

## Motivation and Context


Previously when some select a bucket on the browser, first 1000 objects
of that bucket will be pulled from the server and shown. When the user
clicks on the last modified column to sort the objects, only the first 1000
objects are sorted and shown instead of all the objects from the server.
this will confuse the user. So with the new approach, we will pull all
the objects from the server and sort it based on last modified and show them.
Local pagination is added to browser ui to avoid slowness when there
are lot of objects.

## Regression


No

## How Has This Been Tested?



1. Upload 10k objects to a bucket
2. Check if they are sorted by default on last modified column
3. Click on `Name` column to sort the list based on the object name
4. Scroll down and see if more objects are loaded
5. Test above steps with federation setup

## Types of changes


## Checklist:
